client/src/lib/solana/solid-data.ts

```diff
@@ -2,6 +2,7 @@ import { clusterApiUrl, Cluster, PublicKey } from '@solana/web3.js';
 import { Assignable, Enum, SCHEMA } from './solana-borsh';
 import { DID_METHOD, DID_HEADER } from '../constants';
 import { encode } from 'bs58';
+import { mergeWith } from 'ramda';
```

Comment:
I didn't know about this library before, really neat!

Reply:
probably my favourite JS lib :)